client/main.go

```diff
@@ -316,6 +316,7 @@ func main() {
 		streaming             = flag.Bool("streaming", false, "use the streaming features of strest server")
 		streamingRatio        = flag.String("streamingRatio", "1:1", "the ratio of streaming requests/responses")
 		metricAddr            = flag.String("metric-addr", "", "address to serve metrics on")
+		latencyUnit		      = flag.String("latencyUnit","milli","latency units [milli|micro|nano]")
```

Comment:
It would fit better with our other tools if we stuck with the standard Go `Time.ParseDuration` unit suffixes here: ms, us, ns.